### ISSUE TITLE
Relative URLs with <base> tag not supported

### ISSUE THREAD

**Issue by hlubek** [bslatkin/dpxdt#127]
Some websites use a `<base href="http://www.example.com/" />` tag to specify the base for relative URLs. The site_diff script currently generates wrong URLs in these cases (the relative path is appended to the current URL of the page).

**Comment by bslatkin** [comment]
Good catch! That's a tricky one. What's your expectation of that behavior?

I figure the way it would have to work is to set the prefix here:

https://github.com/bslatkin/dpxdt/blob/master/dpxdt/tools/site_diff.py#L141
